pages/profile/index.js

```diff
@@ -5,21 +5,74 @@ import HeroBanner from 'components/HeroBanner/HeroBanner';
 import Content from 'components/Content/Content';
 import LinkButton from 'components/LinkButton/LinkButton';
 import withAuthSync from 'decorators/withAuthSync/withAuthSync';
+import { getUserPromise } from 'common/constants/api';
+import { boolean } from 'yup';
+import _ from 'lodash';
```

Comment:
Dang! I thought we had a lint rule to prevent this.

Can you please import the specific methods you're using, like so:

`import capitalize from 'lodash/capitalize';`

As it's written now, you'll import the _whole_ lodash library to the client, unnecessarily increasing the bundle size.

I'll write an issue up so we get this lint rule added.